src/Symfony/Component/AssetMapper/Command/ImportMapRequireCommand.php

```diff
@@ -31,10 +33,19 @@ final class ImportMapRequireCommand extends Command
 {
     use VersionProblemCommandTrait;
 
+    private readonly string $projectDir;
+
     public function __construct(
         private readonly ImportMapManager $importMapManager,
         private readonly ImportMapVersionChecker $importMapVersionChecker,
+        /* private readonly string $projectDir, */
```

Comment:
```suggestion
        private readonly ?string $projectDir = null,
```
Then we can `trigger_deprecation()` when it's `null` and make it non nullable in 8.0

Reply:
Done !

Reply:
@chalasr Is their anything else here ?